src/libaudcore/vfs.h

```diff
@@ -66,7 +67,7 @@ constexpr int from_vfs_seek_type(VFSSeekType whence)
                ? SEEK_SET
                : (whence == VFS_SEEK_CUR)
                      ? SEEK_CUR
-                     : (whence == VFS_SEEK_END) ? SEEK_END : -1;
+                     : (whence == VFS_SEEK_END) ? SEEK_END : VFS_SEEK_INVALID;
```

Comment:
This line actually should return -1 still since the return value is an int, not VFSSeekType.

All the other changes LGTM.

Reply:
Right, got confused by the use of the SEEK_CUR/SEEK_END defines!